doc/Quality_white_wine_predictor.md

```diff
@@ -0,0 +1,245 @@
+
+## **EDA first conclusions**
+
+According to our first EDA, we do not have a balanced database, our
+wines are concentrated around quality 5 and 7.5 (around 80% of data
+points). Besides, we have a couple of signs about some variables. For
+instance, it appears that the higher the alcohol level, the better the
+wine quality. Additionally, the smaller the chlorides and total sulphur
+dioxide the better the wine quality. Some variables seem do not
+influence wine quality on their own. When combining these variables,
+they might indeed influence wine quality.
+
+According to our first EDA, we do not have a balanced database, our
+wines are concentrated around quality 5 and 7.5 (around 80% of data
+points). The image below can clear shows up this first finding:
+
+![](../results/Distribution_of_white_wine_quality.PNG)<!-- -->
+
+Besides, we have a couple of signs about some variables. For instance,
+it appears that the higher the alcohol, Sulphates and ph levels the
+better the wine quality. We can check these information on the chart
+below:
+
+![](../results/relationship_between_individual_features_and_the_quality_3.PNG)<!-- -->
+
+Following the same idea, we there are some evidences that the smaller
+the chlorides, Free Sulfur Dioxide, total sulphur dioxide and density
+levels, the better the wine quality. The chart below ilustrates these
+findings:
+
+![](../results/relationship_between_individual_features_and_the_quality_2.PNG)<!-- -->
+
+For some variables like Fixed acidity, Volatile acidity, Citric Acid and
+Residual sugar seem do not influence wine quality on their own. However,
+we need to take care with this statement since when they are combining
+with other variables above, they might indeed influence wine quality.
+The chart below can show up these findings:
+
+![](../results/relationship_between_individual_features_and_the_quality_1.PNG)<!-- -->
+
+## **Cross Validation**
+
+In order to evaluate the best model, we will use the cross validation
+approach to support our decision. The original dataset is partitioned in
+the training set used to train the model, and the test set used to
+predict the values with the trained model. Cross validation partitions
+the training set in the same way and performs the training and
+prediction several times. Then, the result with the best RMSE, accuracy,
+AUC or the chosen metric is selected. This process can be used in
+conjunction to tuning parameters.
+
+# Results & Discussion
+
+## **Evaluation**
+
+After running the models, we used the test-score metrics to evaluate our
+model prediction performance. As we can assess into the table below,
+Model KNN is the best in terms of test-score metrics, with XXX in
+comparison with other models (XXXXX)
+
+![](../results/dummy_table.PNG)<!-- -->
+
+In the context of our business question focusing on the prediction of
+white wine quality, it is reasonable that KNN gives us superior
+“predictions.”
+
+## **Discussion**
```

Comment:
I think that our scores weren't very good and that is worth mentioning in here as well.